doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -7703,3 +7703,29 @@ Parameters:
 
 ``per_page`` Number of results returned per page.
 
+
+Customization File Contents
+---------------------------
+
+The Customization API is used to retrieve the analytics-code.html as well as other customization file contents. See also :ref:`web-analytics-code` in the Configuration section of the Installation Guide.
+
+The content is returned in type="text/html; charset=UTF-8"
```

Comment:
When you pass "logo" I would think the content type should be `image/png` or `image/jpeg` or whatever format the logo is in.

Even for the analytics code, the charset might be something other than UTF-8, right? Should we detect the charset in use and return that?